packages/ra-core/src/controller/create/useCreateController.ts

```diff
@@ -49,20 +51,23 @@ export const useCreateController = <
     props: CreateControllerProps = {}
 ): CreateControllerResult<RecordType> => {
     const {
-        record = {},
-        successMessage,
+        disableAuthentication,
```

Comment:
You should update the Auth doc to mention this prop. Same for other controllers.